.npm-version

```diff
@@ -1 +1 @@
-8.*
\ No newline at end of file
+6.*
```

Comment:
Should be 7.*

Reply:
I believe 7 is not supported in the pipeline so will leave at 6. I will bring this up with the platform team.